proto/ibc/core/channel/v1/tx.proto

```diff
@@ -301,9 +301,10 @@ message MsgChannelUpgradeTry {
 
 // MsgChannelUpgradeTryResponse defines the MsgChannelUpgradeTry response type
 message MsgChannelUpgradeTryResponse {
-  string             channel_id = 1;
-  string             version    = 2;
-  ResponseResultType result     = 3;
+  string             channel_id       = 1;
+  Upgrade            upgrade          = 2 [(gogoproto.nullable) = false];
```

Comment:
similarly to INIT response, upgrade contains version

Reply:
what happened to version?